test/SlowTests/Issues/RavenDB-13553.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.ServerWide.Operations;
+using Raven.Tests.Core.Utils.Entities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_13553 : RavenTestBase
+    {
+        public RavenDB_13553(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task Test()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new User
+                    {
+                        Name = "Yonatan"
+                    });
+
+                    session.SaveChanges();
+                }
+
+                var config = new PeriodicBackupConfiguration
+                {
+                    LocalSettings = new LocalSettings
+                    {
+                        FolderPath = @"C:\RavenBackups"
+                    },
+
+                    //Full Backup period (Cron expression for a 3-hours period)
+                    FullBackupFrequency = "0 */3 * * *",
+
+                    //An incremental-backup will run every 20 minutes (Cron expression)
+                    IncrementalBackupFrequency = "*/20 * * * *",
+
+                    BackupType = BackupType.Backup,
+
+                    Name = "fullBackupTask",
+                };
+
+                var operation = new UpdatePeriodicBackupOperation(config);
+                var result = await store.Maintenance.SendAsync(operation);
+
+                var backupStatus = await store.Maintenance.SendAsync(new StartBackupOperation(true, result.TaskId));
+                var x = await backupStatus.WaitForCompletionAsync(TimeSpan.FromMinutes(5));
```

Comment:
```suggestion
                await backupStatus.WaitForCompletionAsync(TimeSpan.FromMinutes(5));
```

Reply:
I deleted that line, we didn't needed it anymore